docker-compose.auditlogs.yml

```
@@ -18,7 +18,7 @@ services:
             - traefik.enable=true
             - traefik.http.routers.auditlogs.entrypoints=https
             - traefik.http.routers.auditlogs.middlewares=userauth,sec-headers,compression,json-error-responder1,json-error-responder4
-            - traefik.http.routers.auditlogs.rule=PathPrefix(`/api/management/{(v[0-9]+)}/auditlogs`)
+            - traefik.http.routers.auditlogs.rule=PathPrefix(`/api/management/{ver:(v[0-9]+)}/auditlogs`)
```

Can we get rid of the parenthesis?
```suggestion
            - traefik.http.routers.auditlogs.rule=PathPrefix(`/api/management/{ver:v[0-9]+}/auditlogs`)
```